### ISSUE TITLE
can't change grid / filename colors?

### ISSUE THREAD

**Issue by kaolin** [sharkdp/bat#178]
I might just be missing how to do this (but my naive attempts at adding those terms to an existing syntax file and re-init'ing the cache didn't seem to do anything... I haven't played with rust, so believe I might just be missing something) -->

printer.rs — 
```
278         Colors {
279             grid: Fixed(GRID_COLOR).normal(),
280             filename: White.bold(),
281             git_added: Green.normal(),
282             git_removed: Red.normal(),
283             git_modified: Yellow.normal(),
284             line_number: Fixed(LINE_NUMBER_COLOR).normal(),
285         }

```
Are these overridable somehow, currently?

**Comment by sharkdp** [comment]
No, these colors/styles are not overridable at the moment. The filename, git markers, line numbers and grid components are an addition of `bat` and are not part of the Sublime Text syntax highlighting themes.

I definitely think we should make these configurable by the user, but I'm not sure how. Maybe we should actually use some colors that are defined in the syntax highlighting themes (like `foreground`, `comment` etc)?

**Comment by kaolin** [comment]
Using colors defined in the theme sounds best to me. :)

**Comment by sharkdp** [comment]
This has been fixed in [bat 0.5](https://github.com/sharkdp/bat/releases/tag/v0.5.0).
